build.sh

```diff
@@ -18,7 +18,41 @@ if [ ! -d "artifact" ]; then
     mkdir artifact
     chmod -R 755 artifact
 fi
-echo "For build number: ${build_number}"
 
 ## relocate the war file
 mv target/ctools-project-migration*.war artifact/ctools-project-migration.${build_number}.war
+
+function writeEnvironmentVariables {
+    # Values to be used while writing lines into the here document need to be evaluated here.
+    # Values written by the here document can be used when the Makefile itself is run.
+    # Variables values to be used when writting the script must be defined outside of the
+    # here document.  Values explicitly defined in the here document can only be used
+    # at build time, hence the escaping of the $ values.
+    local WEBAPPNAME_value=ctools-project-migration
+    local BUILD_value=${BUILD_NUMBER:-imaginary}
```

Comment:
"Imaginary" is just to make clear that there is no build number since it has not been built on the build server.  (Otherwise BUILD_NUMBER would have a value.)

Adding the server name is great for the big CTools applications.  It can be looked up as part of generating the status page, but 'server' makes a lot less sense in the container / OpenShift world so I'm not planning to put working into that.